pii_check/pii_check_hook.py

```diff
@@ -8,24 +8,25 @@
 from pathlib import Path
 
 
-def get_payload(content, enabled_entity_list):
-    if len(enabled_entity_list) == 0:
-        payload = {
-            "text": content,
-            "link_batch": True,
-            "entity_detection": {
-                "accuracy": "high",
-            },
-        }
-    else:
-        payload = {
-            "text": content,
-            "link_batch": True,
-            "entity_detection": {
-                "accuracy": "high",
-                "entity_types": [{"type": "ENABLE", "value": enabled_entity_list}],
-            },
-        }
+def get_payload(content, enabled_entity_list, blocked_list):
+    payload = {
+        "text": content,
+        "link_batch": True,
+        "entity_detection": {
+            "accuracy": "high",
+        },
+    }
+    
+    if enabled_entity_list:
+        payload["entity_detection"]["entity_types"] = [{"type": "ENABLE", "value": enabled_entity_list}]
+
+    if blocked_list:
+        blocked_list_payload = []
+        for block_pattern in blocked_list:
+            curr_block_item = {"type": "BLOCK", "value": block_pattern}
+            blocked_list_payload.append(curr_block_item)
+        payload["filter"] = blocked_list_payload
```

Comment:
According to the 3.0beta2 docs, this should be `payload["entity_detection"]["filter"]` right? unless it has changed since then